schemas/json/theme.json

```diff
@@ -312,258 +312,270 @@
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/query": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/query-pagination": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/query-pagination-next": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/query-pagination-numbers": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/query-pagination-previous": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/query-title": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/quote": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/rss": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/search": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/separator": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/shortcode": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/site-logo": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/site-tagline": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/site-title": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/social-link": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/social-links": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/spacer": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/table": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/table-of-contents": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/tag-cloud": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/template-part": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/term-description": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/text-columns": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/verse": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/video": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/widget-area": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/legacy-widget": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						},
+						"core/widget-group": {
+							"$ref": "#/definitions/settingsPropertiesComplete"
+						}
+					}
 				},
-				"core/widget-group": {
-					"$ref": "#/definitions/settingsPropertiesComplete"
+				{
+					"patternProperties": {
+						"^[a-z][a-z0-9-]*\/[a-z][a-z0-9-]*$": {
```

Comment:
I wasn't aware that it has to start with a letter 😄 

I guess the best part of working on those schemas is that we are able to clarify what edge cases should be.

Reply:
Yeah agreed. Working on these schemas really helps surface some things that I was previously just not aware of like this :)